### PR TITLE
Create a function to create a friendly field name on field name creation

### PR DESCRIPTION
**Pull request by Miniplop**
## Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [ ] All new and existing tests are passing.

### CODE REVIEW

File: packages/slice-machine/lib/models/common/widgets/Group/index.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ import { AsArray, GroupField } from "./type";
 const Meta = {
   icon: MdPlaylistAdd,
   title: "Group",
-  description: "A Group of Prismic widgets",
+  description: "A Group of Prismic fields",
```
Comment (Heyrows) [inline packages/slice-machine/lib/models/common/widgets/Group/index.ts:14]: haha

File: packages/slice-machine/tests/src/utils/fieldNameCreator.test.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import "@testing-library/jest-dom";
+
+import { expect } from "@jest/globals";
+
+import { createFriendlyFieldNameWithId } from "@src/utils/fieldNameCreator";
+
```
Comment (Heyrows) [inline packages/slice-machine/tests/src/utils/fieldNameCreator.test.ts:6]: OPT: a test to verify that it works with more than 2 words (new object in the array)

**Review by Heyrows — APPROVED** [review]

**Review by Heyrows — CHANGES_REQUESTED** [review]
Found a place you forgot, for a field in a group widget.
`slice-machine/packages/slice-machine/lib/models/common/widgets/Group/ListItem/index.js`
on this file, line 69

File: packages/slice-machine/lib/models/common/widgets/Group/index.ts (hunk 1 above)
Comment (Miniplop) [reply packages/slice-machine/lib/models/common/widgets/Group/index.ts:14]: :p

**Review by Heyrows — APPROVED** [review]

**Pull request merged**